pkg/backingstore/reconciler.go

```diff
@@ -967,6 +967,9 @@ func (r *Reconciler) ReconcilePool() error {
 }
 
 func (r *Reconciler) reconcilePvPool() error {
+	if r.Secret.StringData == nil {
```

Comment:
is it needed now, after you added `SecretResetStringDataFromData` to `ReconcileObject`?

Reply:
Tested it and yes, It is still needed.